main.css

```diff
@@ -0,0 +1,227 @@
+body {
+    font-family: Karla Regular;
+    font-weight: 400;
+    font-size: 14px;
+    margin: 0px;
+}
+
+header {
+    height: 48px;
+}
+
+@media (min-width:768px) {
+    header {
+        height: 56px;
+    }
+}
+
+@media (min-width:1024px) {
+    header {
+        height: 125px;
+    }
+}
+
+.menu {
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0;
+}
+
+@media (min-width:1024px) {
+    .menu {
+        display: none;
+    }
+}
+
+.heading {
+    line-height: 20px;
+    color: #191919;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 10px;
+    width: 100%;
+    position: relative;
+    height: 48px;
+}
+
+@media (min-width:768px) {
+    .heading {
+        height: 56px;
+    }
+}
+
+@media (min-width:1024px) {
+    .heading {
+        line-height: 24px;
+        height: 61px;
+    }
+}
+
+.nav-column {
+    width: 33.33%;
+}
+
+.nav-controls {
+    text-align-last: right;
+}
+
+.nav-option {
+    margin: 14px 56px 0px 0px;
+    text-align: center;
+    position: relative;
+    display: inline-block;
+    cursor: pointer;
+    height: 20px;
+    text-align: center;
+}
+
+@media (min-width:768px) {
+    .nav-option {
+        margin: 16px 56px 0px 0px;
+    }
+}
+
+@media (min-width:1024px) {
+    .nav-option {
+        margin: 24px 16px 0px 0px;
+        height: 24px;
+    }
+}
+
+.create-list {
+    background: #142D3B;
+    color: white;
+    text-transform: uppercase;
+    width: 94px;
+    padding: 4px 16px;
+    margin-right: 32px;
+}
+/* Is this ok to make the breakpoint 1023 instead of 1024? Same for a few media queries below. No opposite for display:none to make it min-width:1024*/

```

Comment:
well, in theory you can create any breakpoint you want but my question is - why 1023? Width <= 1024 is usually a tablet width, so you'd wanna style either <= 1024 for tablet, or >1024 for desktop

Reply:
Have amended it now to use display block on the media query and display none without media query.